content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md

```diff
@@ -0,0 +1,25 @@
+Title: How PyDelhi conducts Hiring and Pitching sessions
+Slug: pydelhi-hiring-pitching-sessions
+Date: 2018-05-02
+Category: General
+Author: Shashank Kumar
+Email: abc@shanky.xyz
+Summary: Hiring and Pitching sessions have been a part of PyDelhi events since the beginning. Here's how it is organized.
+
+![Hiring Pitching Sessions]({filename}/images/hiringpitching.jpg)
+
+Hiring and Pitching sessions have been a part of PyDelhi events since the beginning. The session is scheduled close to the end of every event just before the closing notes. This blog is to help out volunteers and participants by putting organized structure to the entire process. Points raised below are the work of volunteers, open to discussion and will be shaped better with time.
+
+## Step 1 - Broadcasting on social media
+
+The initial step, before even the event start, volunteers should be sharing the link to etherpad which will be used to jot down details of folks speaking during the session. This will help folks to be prepared beforehand and can be called by the volunteers during the session. Also, this will be beneficial for the folks who are not able to attend the event. Official PyDelhi channels (Twitter/FB) should be preferred to broadcast the link. Etherpad link which can be used is https://public.etherpad-mozilla.org/p/Oms9Iw8MK4
```

Comment:
Reworded the first sentence `The initial step, before even the event start, volunteers should be sharing the link to etherpad which will be used to jot down details of folks speaking during the session.`  to `Before the event starts, volunteers should share a link on etherpad to note down the detials of the speakers.`

Reply:
ACK.